tests/wpunit/MediaItemMutationsTest.php

```diff
@@ -756,6 +764,117 @@ public function testCreateMediaItemMutation() {
+					'caption'       => apply_filters( 'the_content', $this->caption ),
+					'commentStatus' => $this->commentStatus,
+					'date'          => $this->date,
+					'dateGmt'       => $this->dateGmt,
+					'slug'          => 'test',
+					'status'        => strtolower( $this->status ),
+					'mimeType'      => 'image/png',
+					'parent'        => null,
+					'mediaType'     => 'image',
+					'sourceUrl'     => $attachment_url,
+					'mediaDetails'  => [
+						'file'   => $attachment_details['file'],
+						'height' => $attachment_details['height'],
+						'meta'   => [
+							'aperture'         => 0.0,
+							'credit'           => '',
+							'camera'           => '',
+							'caption'          => '',
+							'createdTimestamp' => null,
+							'copyright'        => '',
+							'focalLength'      => null,
+							'iso'              => 0,
+							'shutterSpeed'     => null,
+							'title'            => '',
+							'orientation'      => '0',
+						],
+						'width'  => $attachment_details['width'],
+						'sizes'  => [
+							0 => [
+								'name'      => 'medium',
+								'file'      => $attachment_details['sizes']['medium']['file'],
+								'width'     => (int) $attachment_details['sizes']['medium']['width'],
+								'height'    => (int) $attachment_details['sizes']['medium']['height'],
+								'mimeType'  => $attachment_details['sizes']['medium']['mime-type'],
+								'sourceUrl' => wp_get_attachment_image_src( $attachment_id, 'medium' )[0],
+							],
+							1 => [
+								'name'      => 'large',
+								'file'      => $attachment_details['sizes']['large']['file'],
+								'width'     => (int) $attachment_details['sizes']['large']['width'],
+								'height'    => (int) $attachment_details['sizes']['large']['height'],
+								'mimeType'  => $attachment_details['sizes']['large']['mime-type'],
+								'sourceUrl' => wp_get_attachment_image_src( $attachment_id, 'large' )[0],
+							],
+							2 => [
+								'name'      => 'thumbnail',
+								'file'      => $attachment_details['sizes']['thumbnail']['file'],
+								'width'     => (int) $attachment_details['sizes']['thumbnail']['width'],
+								'height'    => (int) $attachment_details['sizes']['thumbnail']['height'],
+								'mimeType'  => $attachment_details['sizes']['thumbnail']['mime-type'],
+								'sourceUrl' => wp_get_attachment_image_src( $attachment_id, 'thumbnail' )[0],
+							],
+							3 => [
+								'name'      => 'medium_large',
+								'file'      => $attachment_details['sizes']['medium_large']['file'],
+								'width'     => (int) $attachment_details['sizes']['medium_large']['width'],
+								'height'    => (int) $attachment_details['sizes']['medium_large']['height'],
+								'mimeType'  => $attachment_details['sizes']['medium_large']['mime-type'],
+								'sourceUrl' => wp_get_attachment_image_src( $attachment_id, 'medium_large' )[0],
+							],
+						],
+					],
+				],
+			],
+		];
+
+		$this->assertEquals( $expected, $actual['data'] );
+	}
```

Comment:
@renatonascalves Maybe use the `assertQuerySuccessful()` assertion, and the `expectField()`, `expectedObject()`, `expectedNode()`, and `expectedEdge()` to strictly test the parts of the query you need to test.

Reply:
Good suggestion! I was trying to follow the same pattern already used in the test file, though.